.github/workflows/autofix.yml

```diff
@@ -0,0 +1,27 @@
+name: autofix.ci
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  # We only do this on PRs to avoid the (admittedly unlikely) scenario that
+  # we run, green, wait, merge, then the build on `main` could fail because conda
+  # has updated during the "green" and then the "build on `main`" steps
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Rebuild dist
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - run: npm install
+      - run: npm i -g @vercel/ncc
+      - run: ncc build index.js
+        env:
+          NODE_OPTIONS: "--openssl-legacy-provider"
+      - uses: autofix-ci/action@551dded8c6cc8a1054039c8bc0b8b48c51dfc6ef
```

Comment:
I would say we should pin the latest 1.3.1 hash as only the readme changed since then.

Reply:
Their official docs suggest to use this hash, so maybe better to follow that?

Reply:
I don't think dependabot would pick up hash changes (without tag changes), so I would open a PR for their readme. (At least following the readme change history I didn't get the notion of them indeed suggesting to tag to individual hashes unassociated with any release)